nim/flatbuffers/src/table.nim

```diff
@@ -2,7 +2,7 @@ import endian
 
 
 type
-   uoffset* = uint32 ## offset in to the buffer
+   uoffset* = uint32 ## offset into the buffer
```

Comment:
changes for `into` have been quite controversial -- I'm happy to drop anything...